debian/changelog

```diff
@@ -0,0 +1,11 @@
+azure-gdpr (1.8.citus-1) stable; urgency=low
+
+  * Official 1.8 release of Azure GDPR
+
+ -- Philip Dube <phdub@microsoft.com>  Mon, 25 Apr 2022 10:45:00 +0300
+
+azure-gdpr (1.7.citus-1) stable; urgency=low
+
+  * Official 1.7 release of Azure GDPR
+
+ -- Philip Dube <phdub@microsoft.com>  Wed, 05 Jan 2022 10:45:00 +0300
```

Comment:
We will not actually create debian packages for 1.7 here. So why do we have an entry here?

Also I see some more tags that have weird versions. Why do we have https://github.com/citusdata/azure_gdpr/releases/tag/v1.21 and https://github.com/citusdata/azure_gdpr/releases/tag/v1.21.0 ?

Reply:
I see that those releases are now removed